templates/security/cve/cve.html

```diff
@@ -209,13 +209,13 @@ <h2>Status</h2>
                           {{ patch.content }}
                         {% elif patch.type == "break-fix" %}
                           Introduced by
-                          <a href="https://git.kernel.org/linus/{{ patch.content.introduced }}" target="_blank">{{ patch.content.introduced }}</a><br>
+                          <a href="https://git.kernel.org/linus/{{ patch.content.introduced }}" target="_blank" rel="noopener noreferrer">{{ patch.content.introduced }}</a><br>
```

Comment:
I hope this is to try and retain users when they have indicated they want to follow a link.